exec-analysis/src/main/java/com/capitalone/dashboard/exec/model/Person.java

```diff
@@ -0,0 +1,211 @@
+package com.capitalone.dashboard.exec.model;
```

Comment:
Is this class only usable in this module? If not this should be in the core.

Reply:
Used only in exec-analysis.